build.sbt

```diff
@@ -654,6 +655,24 @@ lazy val `text-buffer` = project
     )
   )
 
+val generateRustParser = TaskKey[Unit]("generateRustParser", "Generates parser sources")
+val generateRustParserSettings = generateRustParser := {
+    import sys.process._
+    Seq("cargo", "run", "-p", "enso-parser-generate-java", "--bin", "enso-parser-generate-java", "lib/rust/parser/generate-java/java/org/enso/syntax2/") !
+}
+
+lazy val `syntax-rust-definition` = project
+  .in(file("lib/rust/parser"))
```

Comment:
This definition includes #3555 generated classes into `runtime.jar`. Run `sbt buildEngineDistribution` and then:
```bash
enso$ unzip -l runtime.jar | grep org/enso/syntax2 | cut -c31-
org/enso/syntax2/
org/enso/syntax2/serialization/
org/enso/syntax2/Line.class
org/enso/syntax2/MultiSegmentAppSegment.class
org/enso/syntax2/MultipleOperatorError.class
org/enso/syntax2/OperatorBlockExpression.class
org/enso/syntax2/OperatorLine.class
org/enso/syntax2/Parser.class
...
org/enso/syntax2/serialization/Message.class
```